src/splat/util/conf.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any, Dict, List, Optional, Set, Tuple, Union
+from pathlib import Path
+
+# This unused import makes the yaml library faster. don't remove
+import pylibyaml  # pyright: ignore
+import yaml
+
+import sys
+
+from . import log, options, vram_classes
+
+def merge_configs(main_config, additional_config):
+    # Merge rules are simple
+    # For each key in the dictionary
+    # - If list then append to list
+    # - If a dictionary then repeat merge on sub dictionary entries
+    # - Else assume string or number and replace entry
+
+    for curkey in additional_config:
+        if curkey not in main_config:
+            main_config[curkey] = additional_config[curkey]
+        elif type(main_config[curkey]) != type(additional_config[curkey]):
+            log.error(f"Type for key {curkey} in configs does not match")
+        else:
+            # keys exist and match, see if a list to append
+            if type(main_config[curkey]) == list:
+                main_config[curkey] += additional_config[curkey]
+            elif type(main_config[curkey]) == dict:
+                # need to merge sub areas
+                main_config[curkey] = merge_configs(
+                    main_config[curkey], additional_config[curkey]
+                )
+            else:
+                # not a list or dictionary, must be a number or string, overwrite
+                main_config[curkey] = additional_config[curkey]
+
+    return main_config
+
+def resolve_path(base: Path, rel: Path, include_paths: List[Path]) -> Path:
+    if (base / rel).exists():
+        return (base / rel)
+
+    for path in include_paths:
+        candidate = path / rel
+        if candidate.exists():
+            return candidate
+    log.error(f"\"{rel}\" not found")
+    return None
```

Comment:
You should remove this line since `log.error` will halt execution

Reply:
This should be considered a fatal misconfiguration.

Reply:
I was referring to the `return None`, it is dead code. Sorry for the confussion